pkg/sync/v6/data/Role.rbac.authorization.k8s.io/openshift/shared-resource-editor.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "true"
+  name: shared-resource-editor
+  namespace: openshift
+rules:
+- apiGroups:
+  - ""
+  - template.openshift.io
+  resources:
+  - templates
+  verbs:
+  - update
+  - patch
+  - delete
+  - create
+- apiGroups:
+  - ""
+  - image.openshift.io
+  resources:
+  - imagestreamimages
+  - imagestreams
+  - imagestreamtags
+  verbs:
+  - update
+  - patch
+  - delete
+  - create
+- apiGroups:
+  - ""
+  - image.openshift.io
+  resources:
+  - imagestreams/layers
+  verbs:
+  - update
+  - patch
+  - delete
+  - create
+- apiGroups:
+  - ""
+  - project.openshift.io
+  resources:
+  - projects
+  - namespaces
+  verbs:
+  - get
+  - update
```

Comment:
security: are you sure that this doesn't allow a customer admin to patch an arbitrary `openshift-*` project?  Probably an e2e to prove it would be a good plan.

Reply:
Yes,  but its role and its assigned only to Openshift namespace. I think we would be testing RBAC here. But ok if we want to be sure our config is right.

Reply:
TODO: Write e2e test for config